macos.md

```diff
@@ -36,7 +36,7 @@ Make sure that you're running the 2nd-newest macOS version or the newest version
    <img src="./macos-4.1-homebrew-next-steps.png"><br><br>
 6. Copy each line in the following text, paste it in the terminal and hit return.<br><br>
    ```bash
-   brew install flyctl git less node@20 python
+   brew install flyctl git less node@20 python -f
```

Comment:
1. We use long flags (with `--`, full word 
2. It should be at the same position as the other flags that we use, not separate

Reply:
ac2b6b4326d4cf5f5d372d1b920fe2c0e902f263